.github/workflows/windows-utils.yaml

```
@@ -66,4 +66,4 @@ jobs:
             python -m pip install -e .
         - name: Running all Tests
           run: |
-            pytest -vvv -rx --cov=echopype --cov-report=xml --log-cli-level=WARNING --disable-warnings echopype/tests/utils |& tee ci_${{ matrix.python-version }}_test_log.log
+            pytest -k 'not test_raw_to_mvbs' -vvv -rx --cov=echopype --cov-report=xml --log-cli-level=WARNING --disable-warnings echopype/tests/utils |& tee ci_${{ matrix.python-version }}_test_log.log
```

Maybe Windows doesn't like single quotes, so changing to double quotes.
```suggestion
            pytest -k "not test_raw_to_mvbs" -vvv -rx --cov=echopype --cov-report=xml --log-cli-level=WARNING --disable-warnings echopype/tests/utils |& tee ci_${{ matrix.python-version }}_test_log.log
```